packages/xo-web/src/common/intl/messages.js

```diff
@@ -1036,6 +1036,7 @@ const messages = {
   osKernel: 'OS kernel',
   autoPowerOn: 'Auto power on',
   ha: 'HA',
+  nestedVM: 'Nested VM',
```

Comment:
I'm not sure this label is clear enough for users.

@olivierlambert any suggestions?

Reply:
"Nested virtualization" would be preferable

Reply:
And maybe adding a tooltip to explain when this should be used.

Reply:
It's a pretty common thing in the virtualization world. Being an option in a advanced tab should be enough.